### PR TITLE
enable POV at node level

### PR DESCRIPTION
**Pull request by Palmer-JC**
Allow TargetCamera sub-classes, that have no target, as well as Spolight & DirectionalLight to be positioned & "rotated" in amountRight-amountUp-mountForward & flipBack-twirlClockwise-tiltRight terms.

When integrated into the POV replacement extension, QueuedInterpolation, a scene beforerender will be used instead of a mesh beforerender.

### CODE REVIEW

File: src/Cameras/babylon.camera.ts [hunk 1]
```diff
@@ -152,8 +152,9 @@
 
         private _globalPosition = Vector3.Zero();
 
-        constructor(name: string, public position: Vector3, scene: Scene) {
+        constructor(name: string, position: Vector3, scene: Scene) {
```
Comment (deltakosh) [inline src/Cameras/babylon.camera.ts]: why this change? this is the same behavior

File: src/Lights/babylon.directionalLight.ts [hunk 2]
```diff
@@ -16,9 +15,10 @@
         private _orthoTop = Number.MIN_VALUE;
         private _orthoBottom = Number.MAX_VALUE;
 
-        constructor(name: string, public direction: Vector3, scene: Scene) {
+        constructor(name: string, direction: Vector3, scene: Scene) {
```
Comment (deltakosh) [inline src/Lights/babylon.directionalLight.ts]: Same here? Why this change?

**Comment by deltakosh** [comment]
I have to disagree with this changes. I do not want to pollute top level classes with properties required by some children only

**Comment by Palmer-JC** [comment]
I will go back to yesterday repository, EOD. Implement this with code all in extension.
